src/cl/prysm/prysm_launcher.star

```diff
@@ -262,7 +262,6 @@ def get_beacon_config(
         "--slots-per-archive-point={0}".format(32 if constants.ARCHIVE_MODE else 8192),
         "--suggested-fee-recipient=" + constants.VALIDATING_REWARDS_ACCOUNT,
         # Set per Pari's recommendation to reduce noise
```

Comment:
remove the comment too when removing the thing the comment refers to?

Reply:
I have no idea what the comment referred to anymore